introduction.md

```diff
@@ -1 +1 @@
-
# Introduction

Hello to the introduction page.

## Favorite Path

My favorite path is for sure Electives!

```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?